sqlitecluster/SQLite.cpp

```diff
@@ -15,7 +15,7 @@ atomic_flag                         SQLite::_sqliteInitialized = ATOMIC_FLAG_INI
 SLockTimer<recursive_mutex> SQLite::g_commitLock("Commit Lock", SQLite::_commitLock);
 
 SQLite::SQLite(const string& filename, int cacheSize, int autoCheckpoint, int maxJournalSize, int journalTable,
-               int maxRequiredJournalTableID) :
+               int maxRequiredJournalTableID, string synchronous) :
```

Comment:
please use `const string&` as it used by `filename`; this avoids an extraneous copy.  (Candidly, not important here, but best to establish the habit for consistency with the rest of the code.)